docs/word/word-add-ins-core-concepts.md

```diff
@@ -0,0 +1,122 @@
+
+Add-ins that target Word 2016 or later run by passing a function into the **Word.run()** method. The function passed into the **run** method must have a context argument. This [context object](/javascript/api/word/word.requestcontext) is different than the context object you get from the Office object, but it is also used to interact with the Word runtime environment. The context object provides access to the Word JavaScript API object model. The following example shows how to initialize and run a Word add-in by using the **Word.run()** method.
+
+```js
+(function () {
+    "use strict";
+
+    // The initialize event handler must be run on each page to initialize Office JS.
+    // You can add optional custom initialization code that will run after OfficeJS
+    // has initialized.
+    Office.initialize = function (reason) {
+        // The reason object tells how the add-in was initialized. The values can be:
+        // inserted - the add-in was inserted to an open document.
+        // documentOpened - the add-in was already inserted in to the document and the document was opened.
+
+        // Checks for the DOM to load using the jQuery ready function.
+        $(document).ready(function () {
+            // Set your optional initialization code.
+            // You can also load saved settings from the Office object.
+        });
+    };
+
+    // Run a batch operation against the Word JavaScript API object model.
+    // Use the context argument to get access to the Word document.
+    Word.run(function (context) {
+
+        // Create a proxy object for the document.
+        var thisDocument = context.document;
+        // ...
+    })
+})();
+```
+
+### Synchronizing Word documents with Word JavaScript API proxy objects
+
+The Word JavaScript API object model is loosely coupled with the objects in Word. Word JavaScript API objects are proxies for objects in a Word document. Actions taken on proxy objects are not realized in Word until the document state has been synchronized. Conversely, the state of the Word document is not realized in the proxy objects until the document state has been synchronized. To synchronize the document state, you run the **context.sync()** method. The following example creates a proxy body object and a queued command to load the text property on the proxy body object, and uses the **context.sync()** method to synchronize the body of the Word document with the body proxy object.
+
+```js
+// Run a batch operation against the Word object model.
+Word.run(function (context) {
+
+    // Create a proxy object for the document body.
+    // The body object hasn't been set with any property values.
+    var body = context.document.body;
+
+    // Queue a command to load the text property for the proxy document body object.
+    context.load(body, 'text');
```

Comment:
Should we be showing `body.load("text")` instead? That pattern is more common in our snippets.